packages/playground/_stories/main/SplitButton/SplitButton.stories.ts

```diff
@@ -55,11 +55,22 @@ OpeningMenu.decorators = [
 		</ui5-menu>
 	${story()}
 	<script>
-		var splitBtnWithMenuDefaultAction = document.querySelector("ui5-split-button");
-		var menuInSplitBtn = document.getElementById("menuInSplitBtnDefaultAction");
-		splitBtnWithMenuDefaultAction.addEventListener("ui5-arrow-click", function () {
-			menuInSplitBtn.open ? menuInSplitBtn.close() : menuInSplitBtn.showAt(splitBtnWithMenuDefaultAction);
-		});
+	const splitBtn = document.querySelector("ui5-split-button");
```

Comment:
Is the sample working, I can't see an open menu at all

https://github.com/SAP/ui5-webcomponents/assets/15702139/d3079e3e-bae0-4fec-a476-4d3634939bf8

Reply:
In the dedicated section in "Opening Menu", the menu opens but if I use the same sample in the common view ("Docs") as shown above still the menu does not open. Is it only me, does it work for you?